mantidimaging/gui/windows/spectrum_viewer/spectrum_widget.py

```diff
@@ -113,12 +116,12 @@ class SpectrumWidget(QWidget):
     spectrum_plot_widget: SpectrumPlotWidget
     image_widget: SpectrumProjectionWidget
 
-    def __init__(self) -> None:
+    def __init__(self, parent: MainWindowView) -> None:
```

Comment:
Can we avoid calling this `parent` here. It has a specific meaning in Qt, and caused issues before. I think best to call it `main_window`.

Reply:
See https://github.com/mantidproject/mantidimaging/pull/1989

The key bit is not passing it to the Qt widget constructor, but still best to call it `main_window`